package/recalboxOS/recalboxOS.mk

```diff
@@ -3,11 +3,16 @@
 # recalboxos
 #
 ################################################################################
-RECALBOXOS_RELEASE = 4.0.0
+RECALBOXOS_RELEASE = 4.0.1
 RECALBOXOS_SOURCE = $(RECALBOXOS_NAME)-$(RECALBOXOS_RELEASE).zip
 RECALBOXOS_SITE = https://github.com/recalbox/recalbox-os/releases/download/$(RECALBOXOS_RELEASE)
 
-ifeq ($(BR2_PACKAGE_RECALBOXOS_UNSTABLE),y)
+ifeq ($(BR2_PACKAGE_RECALBOXOS_STABLE),y)
+	RECALBOXOS_VERSION = 4.0.0
+	RECALBOXOS_EXTRA_DOWNLOADS = \
+		http://archive.recalbox.com/4/$(BR2_ARCH)/stable/last/boot.tar.xz \
+		http://archive.recalbox.com/4/$(BR2_ARCH)/stable/last/root.tar.xz
+else ifeq ($(BR2_PACKAGE_RECALBOXOS_UNSTABLE),y)
```

Comment:
Extra download is not needed for the stable, as the boot.tar.xz and the root.tar.xz are already included in the zip file.
Following line is enough to replace your block.
ifeq ($(BR2_PACKAGE_RECALBOXOS_UNSTABLE),y)